src/modules/RA/DataLayer/components/tabs/PopupTab/PopupFieldList.js

```diff
@@ -69,21 +66,27 @@ const PopupFieldList = ({ fields, updateTemplate }) => {
     return errors;
   }, [translate]);
 
-  useEffect(() => {
+  const onTitleFieldChange = useCallback(cb => value => {
```

Comment:
Not working ;-) You should create a component for this...

Reply:
Done.